cibuildwheel/macos.py

```diff
@@ -460,7 +459,7 @@ def build(options: Options, tmp_path: Path) -> None:
                     env=env,
                     capture_stdout=True,
                 ).strip()
-                testing_archs: list[Literal["x86_64", "arm64"]]  # noqa: F821
```

Comment:
it looks like ruff [2431](https://github.com/charliermarsh/ruff/pull/2431) may be broken